lib/react_client/react_interop.dart

```diff
@@ -424,6 +424,8 @@ class ReactDartInteropStatics2 implements ReactDartInteropStatics {
 class ComponentStatics<T extends Component> {
   final ComponentFactory<T> componentFactory;
   final Component2 componentInstance;
+
+  /// Required for use by [ReactDartInteropStatics2].
```

Comment:
NOTE: This will need to be updated / removed when @joebingham-wk's error boundary PR goes through, since he's moving all the `Component2`-related stuff into its own `ComponentStatics2`.